### DISCUSSION TITLE
Set Up Object Storage

### DISCUSSION THREAD

**Discussion by ingale726** [juicedata/juicefs#3234] [Q&A] [answered]
os:centos

When I type the code 
`juicefs format --storage mysql --bucket localhost:3306/milvus --access-key root --secret-key root localhost test`
It suggests
object storage: open root:root@localhost:3306/milvus: default addr for network 'localhost:3306' unknown [format.go:427]

When I type the code 
`juicefs format --storage mysql --bucket (localhost:3306)/milvus --access-key root --secret-key Qq@654321 localhost test`
It suggests
-bash: syntax error near unexpected token `('

What is the correct code to configure mysql storage?

**Comment by davies** [comment, marked as answer]
Can you try to quote it?
```
juicefs format --storage mysql --bucket "(localhost:3306)/milvus" --access-key root --secret-key Qq@654321 localhost test

```
